x-pack/plugins/ml/public/application/trained_models/models_management/force_stop_dialog.tsx

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { FC } from 'react';
+import { EuiConfirmModal } from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n/react';
+import type { OverlayStart } from 'kibana/public';
+import type { ModelItem } from './models_list';
+import { toMountPoint } from '../../../../../../../src/plugins/kibana_react/public';
+
+interface ForceStopModelConfirmDialogProps {
+  model: ModelItem;
+  onCancel: () => void;
+  onConfirm: () => void;
+}
+
+export const ForceStopModelConfirmDialog: FC<ForceStopModelConfirmDialogProps> = ({
+  model,
+  onConfirm,
+  onCancel,
+}) => {
+  return (
+    <EuiConfirmModal
+      title={
+        <FormattedMessage
+          id="xpack.ml.trainedModels.modelsList.forceStopDialog.title"
+          defaultMessage="Force stop model {modelId}?"
```

Comment:
I'd suggest removing the term `Force` from this dialog, as it seems like an API level detail that doesn't need to be shown in the UI.

Reply:
@peteharverson FYI the DFA page contains the "Force stop" action https://github.com/elastic/kibana/blob/8126488021b2efd674ea1ddec3c99c24029879f5/x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_management/components/action_stop/stop_action_modal.tsx#L34

Reply:
Changed in https://github.com/elastic/kibana/pull/118563/commits/9ed443322fc32bfb0b8d136bb257ab2f9a60801f